src/dealWizard/dealWizardTypes.ts

```diff
@@ -1,9 +1,18 @@
 import { IWizardState } from "services/WizardService";
 
+export enum WizardType {openProposal, partneredDeal, makeAnOffer}
+
 export interface IBaseWizardStage {
   wizardManager: any;
   wizardState: IWizardState;
 
   activate;
   attached: () => void;
 }
+
+export interface IStageMeta {
+  wizardManager: any;
```

Comment:
```suggestion
export interface IStageMeta<T> {
  wizardManager: T;
```

Dunno if makes sense here.